error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ReactorRulesTestOutput.java

```diff
@@ -641,4 +643,8 @@ Mono<Void> testMonoFromFutureSupplier() {
   Mono<Void> testMonoFromFutureSupplierBoolean() {
     return Mono.fromFuture(() -> CompletableFuture.completedFuture(null), true);
   }
+
+  Flux<Integer> testFluxFromStreamSupplier() {
+    return Flux.fromStream(() -> RandomGenerator.StreamableGenerator.of(1));
```

Comment:
:eyes:

Reply:
Damn, I'm not sure what I was doing here 😬.

Reply:
I think something with auto-complete I'd say, but slipped through.